modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureListView.testcase

```diff
@@ -34,188 +38,172 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-131206. Verify if its possible download envelopes by Envelope Description "
+	@description = "LPS-131206. Verify if its possible download envelopes by Envelope Description "
 	@priority = 3
 	test CanDownloadEnvelopeByEnvelopeDescription {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		DigitalSignature.createEnvelope(
-			addEmail = "test@liferay.com",
-			addFullName = "Recipient Full Name",
-			addMessage = "Email Message",
-			addName = "Test",
-			addSubject = "Email Subject",
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf");
-
-		Button.click(button = "Send");
-
-		Alert.viewSuccessMessageText(successMessage = "Your envelope was created successfully.");
-
-		Navigator.openURL();
+		task ("Given a digital envelope is created") {
```

Comment:
Could you change this task to: "`Given a digital signature envelope is created"`